data_juicer/utils/lazy_loader.py

```diff
@@ -0,0 +1,44 @@
+# Code copied from https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/util/lazy_loader.py # noqa: E501
+"""A LazyLoader class."""
+
+import importlib
+import inspect
+import types
+
+
+class LazyLoader(types.ModuleType):
+    """
+    Lazily import a module, mainly to avoid pulling in large dependencies.
+    `contrib`, and `ffmpeg` are examples of modules that are large and not
+    always needed, and this allows them to only be loaded when they are used.
+    """
+
+    # The lint error here is incorrect.
+    def __init__(self, local_name, name):
+        self._local_name = local_name
+        # get last frame in the stack
+        frame = inspect.currentframe().f_back
```

Comment:
@pan-x-c Please check if this work in Ray?

Reply:
It works